conformance/results/mypy/dataclasses_kwonly.toml

```diff
@@ -1,4 +1,4 @@
-conformant = "Partial"
+conformant = "Pass"
 notes = """
```

Comment:
This "notes" section should also be removed (sorry for not mentioning that explicitly)

Reply:
Ah sorry, I should've noticed that XD